client/components/ProductOverview/ProductOverview.jsx

```diff
@@ -1,17 +1,38 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-import mockProductStyles from '../../../MockData/ProductStyles';
 import ImageGallery from './ImageGallery';
 
-import '../../css/ProductOverviewCSS.css';
+import '../../css/ProductOverviewCSS.scss';
 
 const ProductOverview = () => {
-  const currentStyleIndex = 0;
-  const currentStyle = mockProductStyles.results[currentStyleIndex];
+  const [imageIndex, setImageIndex] = useState(0);
+  const [styleIndex, setStyleIndex] = useState(0);
+  const [currentStyle, setCurrentStyle] = useState(null);
+
+  useEffect(() => {
+    axios.get('/products/48436/styles')
```

Comment:
The product_id probably shouldn't be hard coded here.

Reply:
Refactored, I was initially using that specific product id because it had more than 7 thumbnails that I could use to test the functionality of my thumbnail scroller